src/main/java/spoon/reflect/visitor/ImportScannerImpl.java

```diff
@@ -63,12 +63,10 @@
 
 	@Override
 	public <T> void visitCtInvocation(CtInvocation<T> invocation) {
-		// For a ctinvocation, we don't have to import declaring type
 		enter(invocation);
 		scan(invocation.getAnnotations());
 		scanReferences(invocation.getTypeCasts());
 		scan(invocation.getTarget());
-		scan(invocation.getExecutable());
```

Comment:
@tdurieux static call target (class name) should be processed by the prev line `scan(invocation.getTarget())`, isn't it?